orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/DetermineHealthProvidersTaskSpec.groovy

```diff
@@ -65,6 +76,38 @@ class DetermineHealthProvidersTaskSpec extends Specification {
     [application: "app"]                          | null                   || [:]                                           // no health provider names when an exception is raised
   }
 
+  @Unroll
+  def "should check interestingHealthProviderNames based server group 'interestingHealthProviderNames' label for titus "() {
```

Comment:
Might be easier to follow if this test were extracted into a dedicated test spec for `TitusInterestingHealthProviderNamesSupplier`?

As for `DetermineHealthProvidersTask` ... we really just need a test that asserts that we look for a `Supplier` and if found use whatever it returns. 

We don't care if it's Kubernetes/Amazon/Google, etc.

Thoughts?